deploy/osd-cluster-metadata/10-osd-cluster-metadata.openshift-config.ConfigMap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: osd-cluster-metadata
+  namespace: openshift-config
+data:
+  api.openshift.com/ccs: {{ fromCDLabel "api.openshift.com/ccs" }}
+  api.openshift.com/channel-group: {{ fromCDLabel "api.openshift.com/channel-group" }}
+  api.openshift.com/environment: {{ fromCDLabel "api.openshift.com/environment" }}
+  hive.openshift.io/cluster-platform: {{ fromCDLabel "hive.openshift.io/cluster-platform" }}
+  hive.openshift.io/cluster-region: {{ fromCDLabel "hive.openshift.io/cluster-region" }}
+  hive.openshift.io/version: {{ fromCDLabel "hive.openshift.io/version" }}
```

Comment:
Is this where we should be taking versions from? I would assume PKO takes it from the system and we should depend on that.

Reply:
PKO exposes the Kubernetes API Server version, as reported by the /version API endpoint and the OpenShift Version from the ClusterVersion CR to the templating engine.

Would be interesting to see how the different version properties are changing over time during an upgrade.
I also can't imagine a use-case where the version as seen from hive might be important.

Reply:
quoted the values, trying to fix the pipeline and removed the version keys. I forgot that PKO exposes this info already

Reply:
I agree it would be better to take the version information from the spoke itself. Same goes for anything that's already available there. There will be a delay between when the upgrade completes (per CVO) and when hive updates the labels on the hub CD. See [thread](https://redhat-internal.slack.com/archives/CB53T9ZHQ/p1704199303787309) for discussion around this.